src/addrman.h

```diff
@@ -692,8 +696,7 @@ class CAddrMan
     //! Find an entry.
     CAddrInfo* Find(const CNetAddr& addr, int *pnId = nullptr) EXCLUSIVE_LOCKS_REQUIRED(cs);
 
-    //! find an entry, creating it if necessary.
-    //! nTime and nServices of the found node are updated, if necessary.
+    //! Create a new entry and add it to the internal data structures mapInfo, mapAddr and vRandom.
```

Comment:
Nit: naming the internal data structures (`mapInfo`, `mapAddr` and `vRandom`) is probably too much implementation-specific detail for the header comments:

```suggestion
    //! Create a new entry and add it to the internal data structures.
```

Reply:
I had the same thought but then changed my mind it in order to prevent the misunderstanding that _all_ relevant internal structures (notably `vvNew` and `vvTried`) are populated. This is something the caller has to do.